package.json

```diff
@@ -45,7 +45,10 @@
     "@olympusdao/component-library": "^2.2.2-alpha.16",
     "@rainbow-me/rainbowkit": "^0.5.0",
     "@reduxjs/toolkit": "^1.8.5",
-    "@tanstack/react-query": "^4.2.3",
+    "@tanstack/react-query": "^4.0.10",
```

Comment:
do you mean to revert this back?

Reply:
i just reverted that. somehow it slipped in with all the syncs.